testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveStorageFormats.java

```diff
@@ -573,12 +532,62 @@ public void testStructTimestamps(StorageFormat format)
                     entry.getId()));
         }
 
+        assertStructTimestamps(tableName, TIMESTAMPS_FROM_HIVE);
+        onHive().executeQuery(format("DROP TABLE %s", tableName));
```

Comment:
since the table is created `onPresto()`, it should be also dropped  `onPresto()`
(may not matter here, but we used to have authorization problems in the past; let's keep things simpler)